R/input-utils.R

```diff
@@ -7,7 +7,7 @@ controlLabel <- function(controlName, label) {
 # refers to values. Below is a function for backward compatibility.
```

Comment:
Can you add a comment to the top of this function saying it always coerces to character. And same for `choicesWithNames` (but for leaf nodes there).